wmn-data.json

```diff
@@ -6404,6 +6404,19 @@
         "cat" : "art",
         "valid" : true
        },
+       {
+        "name" : "TikTok",
+        "uri_check" : "https://www.tiktok.com/oembed?url=https://www.tiktok.com/@{account}",
+        "uri_pretty" : "https://www.tiktok.com/@{account}?lang=en",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : "author_url",
+        "m_string" : "Couldn't find this account",

```

Comment:
I'm getting `"message": "Something went wrong",` when I do an account that is not found. Can you recheck this? Also getting `"code": 400`.